basics/patterns examples/1 12 123 1234 pyramid.py

```diff
@@ -0,0 +1,6 @@
+rows=int(input("Type the number of rows needed:"))
+
+for i in range(1,rows+1):
+    for j in range(1,i+1):
+        print(j,end="")
+    print()
```

Comment:
Maybe we could show one triangle program as example code and have the other as an exercise?